test/views/declaration/additionaldeclarationtype/DeclarationTypeViewSpec.scala

```diff
@@ -181,41 +181,42 @@ class DeclarationTypeViewSpec extends UnitViewSpec with ExportsTestData with Com
       }
     }
 
-    "display header with hint" when {
+    "display header" when {
+      val validated = validatedMessages(request)
 
       "used for Standard Declaration journey" in {
 
-        val view = createView(formStandard, DeclarationType.STANDARD)
+        val view = createView(formStandard, DeclarationType.STANDARD, messages = validated)
 
-        view.getElementById("title").text() mustBe messages("declaration.declarationType.header.standard")
+        view.title() must include(validated("declaration.declarationType.header.standard"))
```

Comment:
This reminds me of `containMessage` matcher used in Movements. Might be worth moving over to Declarations.

Reply:
Agree.  But in general I've taken the approach of minimal fixes to the tests as I'm relying on the tests to validate that the refactoring of the views is correct.  If we start refactoring the tests as well (more) mistakes might occur.   (Not in this case, but in general).